engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/formatter/ConfluentAvroMessageFormatter.scala

```diff
@@ -18,13 +18,14 @@ import org.apache.kafka.common.errors.SerializationException
   */
 private[confluent] class ConfluentAvroMessageFormatter(schemaRegistryClient: SchemaRegistryClient) extends AbstractKafkaAvroDeserializer {
 
+  private val encoderFactory = EncoderFactory.get
+
   schemaRegistry = schemaRegistryClient
 
-  private val encoderFactory = EncoderFactory.get
+  def writeTo(data: Array[Byte], output: PrintStream): Unit = {
```

Comment:
Take a look at that: https://github.com/confluentinc/schema-registry/blob/6b550067e034fe03064e00ea1a8f73612080648a/client/src/main/java/io/confluent/kafka/schemaregistry/avro/AvroSchemaUtils.java#L157 

It looks like in new version we can just use one method for all this stuff...

Reply:
You're rigth.. But after I've bumped up version to 5.5 I had problem with flink.. :|  So, can  we do update confluent in futere?

Reply:
Indeed I take a look at wrong version